modules/statics/src/networks.ts

```diff
@@ -169,19 +169,19 @@ class AvalancheP extends Mainnet implements AvalancheNetwork {
   hrp = 'avax';
   alias = 'P';
   vm = 'platformvm';
-  txFee = BigInt(1000000); // 1 MILLIAVAX
-  createSubnetTx = BigInt(1000000000); // 1 AVAX
-  createChainTx = BigInt(1000000000); // 1 AVAX
-  creationTxFee = BigInt(10000000); // 1 CENTIAVAX
+  txFee = 1000000; // 1 MILLIAVAX
+  createSubnetTx = 1000000000; // 1 AVAX
+  createChainTx = 1000000000; // 1 AVAX
+  creationTxFee = 10000000; // 1 CENTIAVAX
   minConsumption = 0.1;
   maxConsumption = 0.12;
-  maxStakingDuration = BigInt(31536000); // 1 year
-  maxSupply = BigInt(720000000) * BigInt(1000000000); // 720 mil tokens
-  minStake = BigInt(1000000000) * BigInt(2000);
+  maxStakingDuration = 31536000; // 1 year
+  maxSupply = 720000000 * 1000000000; // 720 mil tokens
```

Comment:
this exceeds JS number overflow limit and will likely cause you problems if you try to do math with it.
this is `7.2 * 10^17`
safe overflow limit for JS is ~ `9 * 10^15`

Reply:
alternative is to make this a string instead, and call it `maxSupplyString`

Reply:
when you consume it, you can use any BigNumber/BigInt library to safely do math on it.

Reply:
updated it to string